project/Build.scala

```diff
@@ -475,16 +477,6 @@ object ScaldingBuild extends Build {
     }
   ).dependsOn(scaldingCore, scaldingSerialization)
 
-  lazy val scaldingMacros = module("macros").settings(
-    libraryDependencies <++= (scalaVersion) { scalaVersion => Seq(
-      "org.scala-lang" % "scala-library" % scalaVersion,
-      "org.scala-lang" % "scala-reflect" % scalaVersion,
-      "com.twitter" %% "bijection-macros" % bijectionVersion
-    ) ++ (if(isScala210x(scalaVersion)) Seq("org.scalamacros" %% "quasiquotes" % quasiquotesVersion) else Seq())
-  },
-  addCompilerPlugin("org.scalamacros" % "paradise" % paradiseVersion cross CrossVersion.full)
-  ).dependsOn(scaldingCore, scaldingHadoopTest % "test")
```

Comment:
See if the tests all work here? We might need to move a few tests from the macros if it has platform tests over to the scaldingHadoopTest test suite.